src/wallet/walletActions.js

```diff
@@ -0,0 +1,20 @@
+import Promise from 'bluebird';
+
+export const GET_WALLET = '@wallet/GET_WALLET';
+export const GET_WALLET_START = '@wallet/GET_WALLET_START';
+export const GET_WALLET_SUCCESS = '@wallet/GET_WALLET_SUCCESS';
+export const GET_WALLET_ERROR = '@wallet/GET_WALLET_ERROR';
+
+export const getWallet = (username) => {
```

Comment:
How about as this?
export const getWallet = (username) => 
  (dispatch, getState, { steemAPI }) => {
    ...
  }